libs/MeshKernel/include/MeshKernel/Entities.hpp

```diff
@@ -282,3 +310,59 @@ namespace meshkernel
     }
 
 } // namespace meshkernel
+
+inline meshkernel::Point& meshkernel::Point::operator+=(const Point& p)
```

Comment:
Definitions can be moved to the class declaration where the other operators are overloaded.   
Isn't inlining in this case done automatically by the compiler?

Reply:
Like we discussed, the Entities file needs to be refactored.